apps/public-docsite/tsconfig.json

```diff
@@ -14,7 +15,7 @@
     "noImplicitThis": true,
     "noUnusedLocals": true,
     "skipLibCheck": true,
-    "lib": ["es5", "dom"],
+    "lib": ["es2015", "dom"],
```

Comment:
why this change? we are using es2015 features ? I thought this is for v8 which needs to support es5

Reply:
It's to allow async imports, to let webpack to split the icons into a separate bundle. This is just for the website, which does theoretically need to work in IE 11 (it's been broken for awhile now) but we automatically include some polyfills to handle it.